.gitignore

```diff
@@ -29,19 +34,31 @@ bower_components
 # node-waf configuration
 .lock-wscript
 
-# Compiled binary addons (http://nodejs.org/api/addons.html)
-build/Release
+# Compiled binary addons (https://nodejs.org/api/addons.html)
+build/
 
 # Dependency directories
 node_modules/
 jspm_packages/
 
+# TypeScript v1 declaration files
+typings/
```

Comment:
This doesn't seem right? I add custom typings to my typings directory

Reply:
Indeed. This was just what the new GH template includes, but I agree that atm it's just confusing.